tests/tests/test-filter-api.ts

```diff
@@ -126,7 +126,7 @@ describeWithMoonbeam("Moonbeam RPC (EthFilterApi)", `simple-specs.json`, (contex
         data: TEST_CONTRACT_BYTECODE,
         value: "0x00",
         gasPrice: "0x01",
-        gas: "0x4F930",
+        gas: "0xD99C0",
```

Comment:
```suggestion
        // This gas value comes from TODO
        gas: "0xD99C0",
```
Or better yet make it a constant and comment it there. We spent too much time tracking this down to not doc it.